tokio-trace-macros/tokio-trace-macros/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+#![feature(proc_macro_diagnostic)]
```

Comment:
Proc-macros themselves are now stable, right? Can we just enable this feature conditionally so the `tokio-trace-macros` crate can still be used on stable?

Reply:
Creating proc macros is stable of 1.29, but importing them into other crates only stabilizes in 1.30. We might need to stick around with a few nightly-only builds for a little bit.

Reply:
It looks like removing this feature will allow this branch to build successfully on the latest stable compiler (1.30). Can we get rid of it?